e2e/images_test.go

```diff
@@ -0,0 +1,26 @@
+package e2e
+
+import (
+	"path/filepath"
+	"testing"
+
+	"gotest.tools/fs"
+	"gotest.tools/golden"
+	"gotest.tools/icmd"
+)
+
+func TestImageList(t *testing.T) {
+	cmd, cleanup := dockerCli.createTestCmd()
+	defer cleanup()
+	dir := fs.NewDir(t, "")
+	defer dir.Remove()
+
+	cmd.Command = dockerCli.Command("app", "bundle", filepath.Join("testdata", "simple", "simple.dockerapp"), "--tag", "b-simple-app", "--output", dir.Join("simple-bundle.json"))
+	icmd.RunCmd(cmd).Assert(t, icmd.Success)
+	cmd.Command = dockerCli.Command("app", "bundle", filepath.Join("testdata", "simple", "simple.dockerapp"), "--tag", "a-simple-app", "--output", dir.Join("simple-bundle.json"))
```

Comment:
I guess you can remove the `--output` flag, as you don't use it. Just using `--tag` assure you to store the bundle in the bundle store.

Reply:
If I remove this flag then a `bundle.json` file will be crated in the `e2e` folder. With the flag the generated bundle json will be deleted once the test passes.

Reply:
Ok good point!